modules/pel/peltool/peltool.py

```diff
@@ -369,6 +393,8 @@ def main():
         parser.add_argument('-l', '--list',
                             help='List PELs',
                             action='store_true')
+        parser.add_argument('-i', '--id', 
```

Comment:
we could support this option in non-BMC also, right? since we are supporting list option?

Wait for @spinler to share his view.

Reply:
Since we're going to have a nonBMC list, it would make sense to be able to parse a single PEL based on the ID you get back from the list.

I'm not sure how the user would specify the directory though...  because -d is already taken to do something else.
Maybe if it's '-l <dir> -i <id>' then it just parses that PEL?

Reply:
I think we could support all options in non-BMC environment as well by just passing the path so I would suggest some common way to use for all options.

One potential approach that comes to mind is employing an environment variable like PELS_PATH, which users can configure within their environment, thus eliminating the need to manually pass the path for each option.

Reply:
Let's discuss in our next meeting

Reply:
I was working on code to include environment variable but realized it might be simpler as of now without it. If it becomes too complex, I can add environment variable later.